internal/installer/elasticagent_tar_macos.go

```diff
@@ -177,7 +177,7 @@ func (i *elasticAgentTARDarwinPackage) Preinstall(ctx context.Context) error {
 		}
 	}
 
-	srcPath := common.GetElasticAgentWorkingPath(fmt.Sprintf("%s-%s-%s-%s", artifact, downloads.GetSnapshotVersion(common.ElasticAgentVersion), metadata.Os, metadata.Arch))
+	srcPath := common.GetElasticAgentWorkingPath(fmt.Sprintf("%s-%s-%s-%s", artifact, downloads.GetSnapshotVersion(version), metadata.Os, metadata.Arch))
```

Comment:
It was originally https://github.com/elastic/e2e-testing/pull/2354/files#diff-b9eda3d8d7bfbd3b3f3a8c44c7e06743d03ad92a62fe2543eaba3e80d8e34efdR159 but somehow changed in https://github.com/elastic/e2e-testing/pull/2468/files#diff-b9eda3d8d7bfbd3b3f3a8c44c7e06743d03ad92a62fe2543eaba3e80d8e34efdR168

So I think it was a mistake